backend/src/main/java/com/votogether/domain/vote/repository/VoteRepository.java

```diff
@@ -34,4 +34,6 @@ List<VoteStatus> findVoteCountByPostOptionIdGroupByAgeRangeAndGender(
 
     List<Vote> findByMemberAndPostOptionIn(final Member member, final List<PostOption> postOptions);
 
+    List<Vote> findByMember(final Member member);
```

Comment:
`P3`
```suggestion
    List<Vote> findAllByMember(final Member member);
```
목록을 조회하는 JPA 메서드는 `All`을 명시해주는건 어떨까요?!